api/tests/certification/session-management/unit/domain/usecases/get-certification-details_test.js

```diff
@@ -88,32 +70,20 @@ describe('Certification | Session-management | Unit | Domain | UseCases | get-ce
         placementProfileService,
         competenceMarkRepository,
         certificationAssessmentRepository,
-        scoringCertificationService,
         certificationCandidateRepository,
       });
 
       //then
       expect(result).to.be.an.instanceof(CertificationDetails);
       expect(result).to.deep.equal({
-        competencesWithMark: [
-          {
-            areaCode: '1',
-            id: 'recComp1',
-            index: '1.1',
-            name: 'Manger des fruits',
-            obtainedLevel: 1,
-            obtainedScore: 5,
-            positionedLevel: 3,
-            positionedScore: 45,
-          },
-        ],
+        competencesWithMark: [],
         createdAt: certificationAssessment.createdAt,
         completedAt: certificationAssessment.completedAt,
         id: certificationAssessment.certificationCourseId,
         listChallengesAndAnswers: [
           {
             challengeId: 'rec123',
-            competence: '1.1',
+            competence: '',
```

Comment:
on peut avoir une string vide ici ?

Reply:
Bien vu ! Ce test se nomme "should compute the certification details on the fly", ca tombe bien on en veut plus. Je resoud cela en virant le test du coup.

Reply:
Done